sufia-models/app/services/sufia/create_derivatives_service.rb

```diff
@@ -0,0 +1,17 @@
+module Sufia
+  class CreateDerivativesService
+    include Hydra::Derivatives
+
+    def self.run(generic_file)
+      new(generic_file).create_derivatives
```

Comment:
@jcoyne I was under the impression that it is expecting an instance of generic_file as the parameter passed to the initializer. Is that an invalid impression?  If so some comments would be helpful.